.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: java
+
+jdk:
+  - openjdk8
```

Comment:
For pac4j, I use: `oraclejdk8`but I don't think it's an issue...

Reply:
On all our repository Travis fail after Oracle License is changed 😞And now I automatically change `oraclejdk8` to `openjdk8` in all `.travis.yml` files that I touch 😂